test-cases/raft/raft-multidc-parallel-topology-schema-changes-12h.yaml

```diff
@@ -0,0 +1,30 @@
+test_duration: 800
```

Comment:
@aleksbykov I don't think these test cases should be under raft folder.
They are general longevities.
When we add the configuration to enable raft, they test it with raft.